MetaMask Plugin/How to create a MetaMask Plugin

```diff
@@ -0,0 +1,26 @@
+Introduction
+
+1. Metamask plugin what’s new? 
+2. How this will improve or advantages over the extension.
```

Comment:
The guidelines say to not use unnecessary future tense, which in this case means:

```suggestion
2. How this improves the extension.
```

Metamask plugins already exist, so are not in the future, they are already improving Metamask

> grammar lesson over ;)